tsconfig.json

```
@@ -30,6 +30,7 @@
       "@gosling-track": ["./src/gosling-track/index.ts"],
       "@gosling-genomic-axis": ["./src/gosling-genomic-axis/index.ts"],
       "@gosling-brush": ["./src/gosling-brush/index.ts"],
+      "@gosling-dummy": ["./src/dummy-track/index.ts"],
```

Let's change the name. We will eventually use the `@gosling-lang` NPM organization to publish sub-packages, so the convention will be `@gosling-lang/package-name` when we support monorepo.

```suggestion
      "@gosling-lang/dummy-track": ["./src/dummy-track/index.ts"],
```